File: typescript/src/no-2235-add-two-integers.ts
function sum(num1: number, num2: number): number {
  return num1 + num2;
};

// Parse the command line arguments
// NOTE: In JavaScript, the first two arguments are always the path to Node.js and the path to the script being run
const num1 = parseInt(process.argv[2])
const num2 = parseInt(process.argv[3])
console.log(sum(num1, num2));

File: python/src/no_2235_add_two_integers.py
import sys

class Solution:
    def sum(self, num1: int, num2: int) -> int:
        return num1 + num2

# Parse the command line arguments
num1 = int(sys.argv[1])
num2 = int(sys.argv[2])
solution = Solution()
print(solution.sum(num1, num2))